Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG GOLANG_BUILDER=golang:1.19
-ARG OPERATOR_BASE_IMAGE=gcr.io/distroless/static:nonroot
+ARG GOLANG_BUILDER=registry.access.redhat.com/ubi9/go-toolset:1.19
+ARG OPERATOR_BASE_IMAGE=registry.access.redhat.com/ubi9/ubi-minimal:latest
```

Comment:
`ubi-micro` would be better choice I guess. So we plan to use the same imagesd both upstream/downsteam?

Reply:
Ah, didn't know about that, thanks @rabi !

BTW no, i don't think so. But this let us to avoid using different build flags between upstream and downstream.

Reply:
@rabi i had to rollback to `minimal` image. It seems that it fails FIPS test because `openssl` package is needed and isn't installed in it, and since a package manager isn't present in that image, we can't bypass the issue. Too bad.